src/System.Security.Permissions/src/System/ServiceProcess/ServiceControllerPermission.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Permissions;
+
+namespace System.ServiceProcess
+{
+    public sealed class ServiceControllerPermission : ResourcePermissionBase
+    {
+        public ServiceControllerPermission() { }
+        public ServiceControllerPermission(PermissionState state) : base(state) { }
+        public ServiceControllerPermission(ServiceControllerPermissionAccess permissionAccess, string machineName, string serviceName) { }
+        public ServiceControllerPermission(ServiceControllerPermissionEntry[] permissionAccessEntries) { }
+        public ServiceControllerPermissionEntryCollection PermissionEntries { get; }
```

Comment:
Since you are adding a storage for the property, you can also initialize it in the constructor.

Reply:
why do we need to do it ? as we are skipping the implementation completely and we have not done it for other classes like eventlog in this project.

Reply:
Then you should just return null from the property like this:

`public ServiceControllerPermissionEntryCollection PermissionEntries { get => null; }`

Reply:
`public string MyProperty { get; }`, is a short-cut for:

```
private string _myProperty;

public string MyProperty
{
    get
    {
         return _myProperty;
    }
}
```

If you always return null from the property, you do not need the backing field that this shortcut gives you.